product_pack/README.rst

```diff
@@ -0,0 +1,129 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============
+Product Pack
+============
+
+This module extends the functionality of products to support having a
+collection of products in a single one.
+
+Product pack types
+==================
+
+This module defines the following product pack types, that will change how is
+the product shown in the sales order.
+
+Assuming *Pack* is the product pack and *subproducts* are those set to form
+the *pack*, these are your pricing options:
+
+Detailed Component Prices
+-------------------------
+
+This will show 1 line per *subproduct* and 1 line for the *pack*.
+
+All of them will use the price set in the product.
+
+Detailed Totalized Price
+------------------------
+
+This will show 1 line per *subproduct* and 1 line for the *pack*.
+
+The *pack* line will cost the sum of the *subproducts*' prices.
+
+The *subproducts* lines will cost zero.
+
+Detailed Fixed Price
+--------------------
+
+This will show 1 line per *subproduct* and 1 line for the *pack*.
+
+The *pack* line will cost the price set in the product.
+
+The *subproducts* lines will cost zero.
+
+Not Detailed Totalized Price
+----------------------------
+
+This will show 1 line for the *pack*.
+
+The *pack* line will cost the sum of the *subproducts*' prices.
+
+Not Detailed Assisted Price
+---------------------------
+
+This will show 1 line for the *pack*.
+
+The *pack* line will cost the sum of the *subproducts*' prices, and will have a
+button for editing its details.
```

Comment:
This must be improved. If we don't want detailed info then we have only 2 options:

1. Price of the sum of the subproducts -> "Not detailed totalized price"
2. Price of the pack

**assisted price** is an addtional option in both cases (1) and (2)